tests/integ/test_sagemaker_steps.py

```diff
@@ -179,6 +179,96 @@ def test_transform_step(trained_estimator, sfn_client, sfn_role_arn):
         state_machine_delete_wait(sfn_client, workflow.state_machine_arn)
         # End of Cleanup
 
+
+def test_transform_step_with_placeholder(trained_estimator, sfn_client, sfn_role_arn):
+    # Create transformer from previously created estimator
```

Comment:
```suggestion
    # Create transformer from supplied estimator
```

nit: also not really sure this comment really adds much. it's an parameter to your method so I think the "previously created" part is not really necessary...